src/main/java/ca/corefacility/bioinformatics/irida/repositories/filesystem/IridaFileStorageAwsUtilityImpl.java

```diff
@@ -55,11 +60,46 @@ public IridaFileStorageAwsUtilityImpl(String bucketName, String bucketRegion, St
 	 */
 	@Override
 	public IridaTemporaryFile getTemporaryFile(Path file) {
+		String perm = "rwxrwxr-x";
 		try {
 			logger.trace("Getting file from aws s3 [" + file.toString() + "]");
 			Path tempDirectory = Files.createTempDirectory("aws-tmp-");
 			Path tempFile = tempDirectory.resolve(file.getFileName()
 					.toString());
+			Set<PosixFilePermission> permissions = PosixFilePermissions.fromString(perm);
```

Comment:
This has been removed. It was used for galaxy to be able to read/execute the file but we now upload the file to galaxy instead of linking to the files when using an object store